repos/system_upgrade/el7toel8/actors/checkbrltty/actor.py

```diff
@@ -0,0 +1,48 @@
+from leapp.actors import Actor
+from leapp.libraries.actor import library
+from leapp.models import InstalledRedHatSignedRPM, BrlttyMigrationDecision
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+from leapp.reporting import Report
+from leapp.libraries.common.reporting import report_with_remediation, report_generic
+
+
+class CheckBrltty(Actor):
+    """
+    Check if brltty is installed, check whether configuration update is needed.
+    """
+
+    name = 'check_brltty'
+    consumes = (InstalledRedHatSignedRPM,)
+    produces = (Report, BrlttyMigrationDecision,)
+    tags = (ChecksPhaseTag, IPUWorkflowTag)
+
+    def process(self):
+        for fact in self.consume(InstalledRedHatSignedRPM):
+            for rpm in fact.items:
+                if rpm.name == 'brltty':
+                    report_with_remediation(
+                        title='Brltty has incompatible changes in the next major version',
+                        summary='The --message-delay brltty option has been renamed to --message-timeout.\n'
+                                'The -U [--update-interval=] brltty option has been removed.',
+                        remediation='Please update your scripts to be compatible with the changes.',
+                        severity='low')
+                    (migrate_file, migrate_bt, migrate_espeak) = library.check_for_unsupported_cfg()
+                    report_summary = ''
+                    if migrate_bt:
```

Comment:
Wondering if having report_summary = None, checking for not report_summary and using += for string concatenation is more readable
```
report_summary = None
if migrate_bt:
    report_summary = 'Unsupported aliases message'
if migrate_espeak:
    if report_summary:
        report_summary += '\n'
    report_summary += 'eSpeak speech driver message'
...
```

Reply:
Fixed.

Reply:
In fact I had to keep `report_summary = ''`, because otherwise I got:
```
TypeError: unsupported operand type(s) for +=: 'NoneType' and 'str'
```